### PR TITLE
Custom Author Model

### PR DESCRIPTION
**Pull request by KenRLi**
## Completed
- **Custom Author model with**
    - displayName (Username)
    - _password_
    - email
    - first_name
    - last_name
    - id
    - host
    - url
    - github
- **Frontend**
    - home (currently used for debugging)
    - register
    - login
    - update
    - logout

## Todo
Still need to
- Document user model (UML)
- Write tests
- Define and document API endpoints
- Implement API functionality

## Also need to implement
- Post model
- Comment model

## Sidenote
- Creating the author model from scratch took longer than expected
- Want to merge what I have to boost morale

### CODE REVIEW

**Review by JoePotentier — APPROVED** [review]

**Comment by JoePotentier** [comment]
Going to wait for at least one more review and give time for everyone to read this. Great work Ken! Sorry, I've been absent, I have a 2 midterms this week and was away and working all of the weekdays last week.

**Review by ColonelSanders21 — APPROVED** [review]

**Comment by KenRLi** [comment]
Issue: Admin site, Author passwords are hashed but in a text field.

Not major will fix in next push

**Pull request merged**
